src/index.ts

```diff
@@ -43,18 +83,28 @@ export function createIPXHandler ({
       if (params.error) {
         return {
           statusCode: 400,
-          body: params.error
+          body: params.error,
+          headers: plainText
         }
       }
       id = params.id
       modifiers = params.modifiers
     }
 
-    const requestHeaders: Record<string, string> = {}
+    const requestHeaders: Record<string, string> = {
+      [SUBREQUEST_HEADER]: '1'
+    }
     const isLocal = !id.startsWith('http://') && !id.startsWith('https://')
     if (isLocal) {
       const url = new URL(event.rawUrl)
       url.pathname = id
+      if (localPrefix && !url.pathname.startsWith(localPrefix)) {
```

Comment:
We're giving the option to pass in a `localPrefix` to IPX for scenarios like Next.js that have a dedicated folder for static images correct?

Reply:
Yesm exactly